src/devices/DCMotor/samples/DCMotor.sample.csproj

```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Device.Gpio" Version="$(SystemDeviceGpioPackageVersion)" />
-    <!-- Normally you will use following package reference instead  -->
-    <!-- <PackageReference Include="IoT.Device.Bindings" Version="$(IotDeviceBindingsPackageVersion)" /> -->
+    <ProjectReference Include="..\..\..\System.Device.Gpio/System.Device.Gpio.csproj">
```

Comment:
can you fix `/` and `\` here @joperezr since you're fixing merge conflicts?

Reply:
Yup I'll take care of it.